src/ft2font.h

```diff
@@ -18,8 +18,8 @@ extern "C" {
 /*
  By definition, FT_FIXED as 2 16bit values stored in a single long.
  */
-#define FIXED_MAJOR(val) (long)((val & 0xffff000) >> 16)
-#define FIXED_MINOR(val) (long)(val & 0xffff)
+#define FIXED_MAJOR(val) (unsigned short)((val & 0xffff000) >> 16)
```

Comment:
They extract the two 16-bit values. However, there's a bug in `FIXED_MAJOR`; it masks the input as if it were 16.12 (with 4 undefined bits) yet shifts by 16. The top 4 bits get lost.

In the end though, it turns out we never use these values for anything (they're all versions specifiers.)